src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -591,3 +646,288 @@ RouterExecutorEnd(QueryDesc *queryDesc)
 	queryDesc->estate = NULL;
 	queryDesc->totaltime = NULL;
 }
+
+/*
```

Comment:
Missing newline (same in a number of additional places). I suggest running make indent, after installing the citus tools stuff & uncrustify.